### ISSUE TITLE
Weird looking "duplicate" menu for start file without toctree

### ISSUE THREAD

**Issue by sypets** [t3docs/docker-render-documentation#64]
## Reproduce

Render documentation using single file documentation README.md (e.g. from "seeder" extension) with latest Docker 2.1.0 image:

```
git clone https://github.com/TildBJ/seeder
source <(docker run --rm t3docs/render-documentation:v2.1.0 show-shell-commands)
dockrun_t3rd makehtml
open Documentation-GENERATED-temp/Result/project/0.0.0/Index.html
```

## Result

* The main menu is in submenu "seeder"
* additionally, the same menu is on top level, but dimmed

![seeder](https://user-images.githubusercontent.com/13206455/59270547-058f6000-8c52-11e9-857a-415260f8327f.png)

![seeder2](https://user-images.githubusercontent.com/13206455/59270550-09bb7d80-8c52-11e9-96d3-dadb79296b3a.png)

## Related

* #31

**Comment by sypets** [comment]
I tested some more: AFAIK, this problem occurs if the start page of the documentation (doesn't matter what it is, it can be README.rst or Documentation/Index.rst):

* does **not contain a toctree**
* and header level 1 **only has one entry**

## Examples where problem occurs:

1. one file without toctree:

```
=====
Title
=====

Some stuff on start page

Topic 1
=======

sfsd sdfsf

Topic 2
=======

sdfsdf

```


## Examples where problem does not occur:

1. Start page with toctree

```
=====
Title
=====

.. toctree::

   Topic1
   Topic2

```

2. Start page with header level 1 with more than 1 entry

This basically means we leave out the title header. 

```
Topic 1
=======

sfsd sdfsf

Topic 2
=======

sdfsdf

```

**Comment by sypets** [comment]
@marble It would be great, if we could get a timeframe for a solution. Because question is if we can solve this in the near future or we should propose a workaround (e.g. use example 3 above).

**Comment by marble** [comment]
Fixed with Release v2.2.0 (June 14, 2019) = just now
Build t3docs/render-documentation:develop can already be pulled. Other builds are on the way. Looks like
![image](https://user-images.githubusercontent.com/307057/59507402-3fa76e80-8eab-11e9-83b2-a6d1deacfe7f.png)

**Comment by sypets** [comment]
@marble I get errors with the "develop" version. I tested with [timymce](https://gitlab.sgalinski.de/typo3/tinymce) extension. This rendered ok with previous version (except for missing title because of missing Settings.cfg). 

```
git clone https://gitlab.sgalinski.de/typo3/tinymce.git
source <(docker run --rm t3docs/render-documentation:develop show-shell-commands)
dockrun_t3rd makehtml

```

**Comment by marble** [comment]
Yes, thanks for reporting, stupid logic error. I've already published a bugfix version v2.2.1 = master = develop. Should be assembled by docker hub soon.

**Comment by sypets** [comment]
There are still some issues with 2.2.6 + single files.

If you use a doc title and a single file with 2.2.6, you get problems.

# Example 1: Documenation/Index.rst

![menu](https://user-images.githubusercontent.com/13206455/60040036-61c59b80-96b8-11e9-9b91-62072eb281be.png)
